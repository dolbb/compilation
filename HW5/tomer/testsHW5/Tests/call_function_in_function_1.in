int func3(int a){
	if(a > 3){
		print("a>3\n");
		return 32;
	}
	else{
		print("correct_3\n");
		return 22;
	}
}

int func2(int a){
	if(a> 4){
		print("a>4\n");
		return 13;
	}
	else{
		print("correct_2\n");
		return 12;
	}
}


int func1(int a){
	if(a> 15){
		print("a>5\n");
		return 3;
	}
	else{
		print("correct_1\n");
		return 2*a;
	}
}

void main(){

	printi(func1(func2(func3(4))));
	print("\n");
}