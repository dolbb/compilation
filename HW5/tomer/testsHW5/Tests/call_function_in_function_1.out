Loaded: ./exceptions.s
a>3
a>4
correct_1
26
